.github/header-checker-lint.yml

```diff
@@ -1,6 +1,6 @@
 {"allowedCopyrightHolders": [],
  "allowedLicenses": ["Apache-2.0", "MIT", "BSD-3"],
- "ignoreFiles": ["requirements.txt", "requirements-test.txt"],
+ "ignoreFiles": ["**/requirements.txt", "**/requirements-test.txt"],
```

Comment:
Do we also need `**/**/requirements.txt` ?  see: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/4968/checks?check_run_id=1705777265

Reply:
Not sure of the rules, but normally I'd expect **/something to match "something" in any lower directory.